github/repos.go

```diff
@@ -293,7 +293,7 @@ func (s *RepositoriesService) Get(ctx context.Context, owner, repo string) (*Rep
 
 	// TODO: remove custom Accept header when the license support fully launches
 	// https://developer.github.com/v3/licenses/#get-a-repositorys-license
-	acceptHeaders := []string{mediaTypeLicensesPreview, mediaTypeSquashPreview}
+	acceptHeaders := []string{mediaTypeLicensesPreview}
 	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
```

Comment:
Given there's only one accept header value being set now, this can be simplified to just:

```Go
req.Header.Set("Accept", mediaTypeLicensesPreview)
```

How do you feel about that?